install_config/aggregate_logging.adoc

```diff
@@ -855,7 +824,7 @@ still expects to read logs generated using the `json-file` log driver. This
 results in a lack of log ingestion. If this has happened within your logging cluster,
 xref:fluentd-upgrade-source[troubleshoot] it.
 
-Aggregated logging is only supported using the `journald` driver in Docker. See
+Aggregated logging is only supported using the `JSON` driver in Docker. See
```

Comment:
It support both journald and json driver.

Reply:
Perhaps if we rewrite this as, "Aggregated logging is only supported using the `json-file` logging driver with Docker."

Reply:
@portante Aggregated logging haven't drop support to journal logging driver with Docker.  I think we just suggest using json-file.

Reply:
@anpingli, we actually don't support logging application logs through journald, even though we have all the code in place.  There are many many issues with it, and while it is a valid option, we need to correct the docs for 3.9 to rescind support before we release.  We can say that the journald logging driver is an option that is still available to users for compatibility, but in 3.10 or later, we'll be removing support for it entirely.